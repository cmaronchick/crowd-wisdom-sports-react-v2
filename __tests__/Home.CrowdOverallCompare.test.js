import React from 'react';
import CrowdOverallCompare from '../src/components/Home.CrowdOverallCompare'
import renderer from 'react-test-renderer'


describe('it should have a sport, year, season, and week', () => {
    
});

test('should receive the leaderboard in proper format', async () => {
        
    const crowdCompare = renderer.create(<CrowdOverallCompare />).toJSON()
    expect(crowdCompare).toMatchSnapshot()
})

