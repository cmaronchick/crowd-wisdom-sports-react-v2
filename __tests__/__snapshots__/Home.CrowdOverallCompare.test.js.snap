// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should receive the leaderboard in proper format 1`] = `
<div>
  Loading Crowd Results ...
</div>
`;
